terminal-tui/examples/demo/app.rs

```diff
@@ -22,31 +20,40 @@ pub enum Message {
 }
 
 /// All components known to the application.
+#[derive(Debug, Eq, PartialEq, Clone, Hash)]
+pub enum TabId {
+    Welcome,
+    List,
+}
```

Comment:
`TabContainer::child(title: impl ToString, ...)`
`impl Display for TabId`

`container.child(TabId::Welcome, WelcomeScreen::new())`